cmd/lotus-wallet/main.go

```diff
@@ -91,15 +180,38 @@ To configure your lotus node to use a remote wallet:
 var getApiKeyCmd = &cli.Command{
 	Name:  "get-api-key",
 	Usage: "Generate API Key",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "rules",
+			Usage: "filtering rule object (see 'lotus-wallet --help')",
+		},
+	},
```

Comment:
There should also be a flag to accept rules from a file to avoid issues around quoting json as a flag.